packages/eslint-config-eventbrite-react/rules/env.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    env: {
```

Comment:
How about you just stick this at the top of `react.js` file kinda like what we did here: https://github.com/eventbrite/javascript/blob/d50a588bd4e71d4d484fe5415c1fae0165f6c133/packages/eslint-config-eventbrite-legacy/rules/node.js#L2-L4